app/core/models.py

```diff
@@ -215,3 +215,19 @@ class Meta:
 
     def __str__(self):
         return f"{self.faq} viewed at {self.created_at.strftime('%b %d %Y %H:%M:%S')}"
+
+
+class Location(AbstractBaseModel):
+    """
+    Location for event
+    """
+    name = models.CharField(max_length=255, unique=True, verbose_name='Location name')
+    address_line_1 = models.CharField(max_length=255, unique=False)
+    address_line_2 = models.CharField(max_length=255, unique=False)
+    city = models.CharField(max_length=100, unique=False)
+    state = models.CharField(max_length=2, unique=False)
+    zip = models.CharField(max_length=10, unique=False)
```

Comment:
The linter didn't like this one because it's overriding a python builtin. You can simply change it to zipcode. And maybe add more code to make it look like "zip" to the outside if that's what it should be.